### PR TITLE
feature(component): in page navigation collapsed

### PR DESCRIPTION
**Pull request by codeuri**
Navigation collapsed with auto change by section

Ticket: [#78](https://app.zenhub.com/workspaces/ffam-prototype-5c615588801d5a103031639c/issues/unimelb/ffam-prototype/78)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit a682ddf7d3ed99893b6f2c54d88715dc1e722a09

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cb85c61dd30a2000ab25995

**Comment by gepifanio** [comment]
Added review to the ticket

**Comment by gepifanio** [comment]
@codeuri Could you fix the conflicts?

File: components/page/navigation/index.css [hunk 1]
```diff
@@ -35,6 +35,28 @@
...
+    align-items: center;
+    justify-content: center;
+    align-self: center;
+    position: fixed;
+    top: -50%;
+    left: 0;
+    transition: all .3s;
+    padding: .5rem;
+    width: 100%;
+    background: #fff;
```
Comment (gepifanio) [inline components/page/navigation/index.css]: should use vars.css

Comment (TimMansell) [reply components/page/navigation/index.css]: will rectify in issue #1070

**Pull request merged**